sw/device/silicon_creator/lib/drivers/spi_device.c

```diff
@@ -6,18 +6,469 @@
+  X( 7,  0, 0x0c)
+
+/**
+ * BFPT 9th Word
+ * -------------
+ * [31: 0]: Erase type 4 and 3 (not supported, 0x0)
+ */
+#define BFPT_WORD_9(X) \
+  X(31,  0, kBfptNotSupported)
+
+/**
+ * BFPT 10th Word
+ * --------------
+ * [31:11]: Erase 4,3,2 typical time (not supported, 0x0)
+ * [10: 9]: Erase type 1 time unit (ms, 0x0)
+ * [ 8: 4]: Erase type 1 time count, zero-based (0x1d)
+ * [ 3: 0]: Max erase time multiplier, zero-based (0x6)
+ *          formula: 2 * (multiplier + 1) * erase_time
+ */
+#define BFPT_WORD_10(X) \
+  X(31, 11, kBfptNotSupported) & \
+  X(10,  9, 0x0) & \
+  X( 8,  4, 0x1d) & \
+  X( 3,  0, 0x6)
+
+/**
+ * BFPT 11th Word
+ * --------------
+ * [31:31]: Reserved
+ * [30:29]: Chip erase time units (256 ms, 0x1)
+ * [28:24]: Chip erase time count, zero-based (0x3)
+ * [23:23]: Additional byte program time units (us, 0x0)
+ * [22:19]: Additional byte program time count, zero-based (0x0)
+ * [18:18]: First byte program time unit (8 us, 0x1)
+ * [17:14]: First byte program time count, zero-based (0x3)
+ * [13:13]: Page program time unit (8 us, 0x0)
+ * [12: 8]: Page program time count, zero-based (0x1f)
+ * [ 7: 4]: Page size, 2^N (0x8)
+ * [ 3: 0]: Max program time multiplier, zero-based (0x1)
+ */
+#define BFPT_WORD_11(X) \
+ X(30, 29, 0x1) & \
+ X(28, 24, 0x3) & \
+ X(23, 23, 0x0) & \
+ X(22, 19, 0x0) & \
+ X(18, 18, 0x1) & \
+ X(17, 14, 0x3) & \
+ X(13, 13, 0x0) & \
+ X(12,  8, 0x1f) & \
+ X( 7,  4, 0x8) & \
+ X( 3,  0, 0x1)
+
+/**
+ * BFPT 12th Word
+ * --------------
+ * [31: 9]: Erase/program suspend/resume (not supported, 0x0)
+ * [ 8: 8]: Reserved
+ * [ 7: 0]: Prohibited ops during suspend (not supported, 0x0)
+ */
+#define BFPT_WORD_12(X) \
+ X(31,  9, kBfptNotSupported) & \
+ X( 7,  0, kBfptNotSupported)
+
+/**
+ * BFPT 13th Word
+ * --------------
+ * [31: 0]: Erase/program suspend/resume instructions (not supported, 0x0)
+ */
+#define BFPT_WORD_13(X) \
+ X(31, 0, kBfptNotSupported)
+
+/**
+ * BFPT 14th Word
+ * --------------
+ * [31:31]: Deep powerdown support (not supported, 0x1)
+ * [30: 8]: Deep powerdown instructions and delay (not supported, 0x0)
+ * [ 7: 2]: Busy polling (bit 0 using 0x05 instruction, 0x1)
+ * [ 1: 0]: Reserved
+ */
+#define BFPT_WORD_14(X) \
+ X(31, 31, 0x1) & \
+ X(30,  8, kBfptNotSupported) & \
+ X( 7,  2, 0x1)
+
+/**
+ * BFPT 15th Word
+ * --------------
+ * [31:24]: Reserved
+ * [23: 0]: Hold, QE, (4S-4S-4S), 0-4-4 (not supported, 0x0)
+ */
+#define BFPT_WORD_15(X) \
+ X(23,  0, kBfptNotSupported)
+
+/**
+ * BFPT 16th Word
+ * --------------
+ * [31:14]: 4-Byte addressing (not supported, 0x0)
+ * [13: 8]: Soft-reset (instruction 0xf0, 0x80)
+ * [ 7: 7]: Reserved
+ * [ 6: 0]: Status register (read-only, 0x0)
+ */
+#define BFPT_WORD_16(X) \
+  X(31, 14, kBfptNotSupported) & \
+  X(13,  8, 0x80) & \
+  X( 6,  0, 0x0)
+
+/**
+ * BFPT 17th Word
+ * --------------
+ * [31:  0]: Fast read (1S-8S-8S) (1S-1S-8S) (not supported, 0x0)
+ */
+#define BFPT_WORD_17(X) \
+  X(31,  0, kBfptNotSupported)
```

Comment:
nit: Given the non-support of anything past word 16, one wonders if we should find a JEDEC parameter table version that doesn't include words [17..23] and use that version and save a bit of parameter space.

Reply:
I agree. I'll try to find something from a credible source when I get the chance but jedec.org is not very helpful when it comes to previous versions of the specs.